client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "client",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Let's leave the client version as-is. We don't have a release process around the client application yet, so I'd like to leave the version number unchanged until we do.

Reply:
Sounds good